src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Misc.cs

```diff
@@ -1291,5 +1291,20 @@ public void TFS_470020()
             CError.Compare(errorCount, 1, "Error Count mismatch!");
             return;
         }
+
+        [Fact]
+        public void Issue_60543()
+        {        
+            string xsd = Path.Combine(TestData._Root, "issue60543.xsd");
+            XmlSchemaSet s = new XmlSchemaSet();
+            s.ValidationEventHandler += new ValidationEventHandler(ValidationCallback);
+            s.XmlResolver = new XmlUrlResolver();
+            XmlReader r = XmlReader.Create(xsd);
+            s.Add(null, r);
+            s.Compile();
```

Comment:
I'm seeing you've created a schema set but not seeing any validation happening here... See example here https://docs.microsoft.com/en-us/dotnet/api/system.xml.schema.xmlschemaset?view=net-5.0 for usage example (create another XmlReader and pass SchemaSet in the settings, then read the whole doc with a simple loop)

Reply:
This is a positive test. If you will run it at net5 it will fail

Reply:
Asserts validation works inside XmlSchemaSet.Compile() in this case, and not in the validated XmlReader.

Reply:
I have added the validated reader tests, too